wordpress-mobile/WordPress-iOS
# UL&S Tracking: fix tracking issues in the WPcom and Magic link flows.

WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/468

This should fix all of the remaining issues in the Magic Links login tracking.

## Testing:

1. Press “Continue with WordPress.com”
2. Enter an email for an account with a password set.
3. Tap “Continue”

Make sure we did NOT track a `password_challenge` step here, but instead:

🔵 Tracked: unified_login_interaction <click: submit, flow: wordpress_com, source: default, step: start>
🔵 Tracked: unified_login_step <flow: login_password, source: default, step: start>

4. Tap “Get a login link by email”

🔵 Tracked: unified_login_interaction <click: request_magic_link, flow: login_password, source: default, step: start>
🔵 Tracked: unified_login_step <flow: login_magic_link, source: default, step: magic_link_requested>

5. Tap “Open Email”

🔵 Tracked: unified_login_interaction <click: open_email_client, flow: login_magic_link, source: default, step: magic_link_requested>
🔵 Tracked: unified_login_step <flow: login_magic_link, source: default, step: email_opened>

As nothing has been modified beyond this point there’s no need to test further.

## PR submission checklist:
